test/Microsoft.TestPlatform.AcceptanceTests/ArgumentProcessorTests.cs

```diff
@@ -28,5 +28,29 @@ public void PassingNoArgumentsToVsTestConsoleShouldPrintHelpMessage(RunnerInfo r
             //Check for help examples text
             this.StdOutputContains("To run tests: >vstest.console.exe tests.dll");
         }
+
+        [CustomDataTestMethod]
+        [NETFullTargetFramework]
+        [NETCORETargetFramework]
```

Comment:
Since this code doesn't depends on Target framework, can we use only one attribute `NETFullTargetFramework` ?